### PR TITLE
PLAT-1152 | Onboarding for ldek

### PR DESCRIPTION
**Pull request by kvas-damian**

### CODE REVIEW

File: resources/assets/js/components/onboarding/Onboarding.vue [hunk 1]
```diff
@@ -16,6 +16,7 @@
 			/>
 			<component
 				:is="currentStep.component"
+				:messagesSet="messagesSet"
```
Comment (kvas-damian) [inline resources/assets/js/components/onboarding/Onboarding.vue]: `messagesSetKey`?

File: resources/assets/js/components/onboarding/Onboarding.vue [hunk 2]
```diff
@@ -211,9 +212,9 @@ export default {
 				{
 					name: ONBOARDING_STEPS.FINAL,
 					component: WnlOnboardingScreenFinal,
-					text: 'Powitanie',
+					text: 'Zaczynamy!',
 					linkTo: { name: 'onboarding', params: { step: ONBOARDING_STEPS.FINAL } },
-					buttonText: 'Otwórz Wstępny LEK',
+					buttonText: 'lastStepConfirm',
```
Comment (kvas-damian) [inline resources/assets/js/components/onboarding/Onboarding.vue]: `buttonTextKey`

File: resources/assets/js/components/onboarding/OnboardingScreenTutorial.vue [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <template>
 	<div class="has-text-centered">
 		<h2 class="title">Obsługa platformy 👨🏻‍🏫</h2>
-		<p class="title is-5">Zanim wylądujesz na Dashboardzie, zapoznaj się z nim w 3 minuty!</p>
+		<p class="title is-5">Zanim wylądujesz na Dashboardzie, zapoznaj się z nim w 2 minuty!</p>
 		<div class="onboarding-video-wrapper margin-top-huge">
 			<iframe
```
Comment (kvas-damian) [inline resources/assets/js/components/onboarding/OnboardingScreenTutorial.vue]: Are we changing the video

File: resources/assets/js/components/onboarding/OnboardingScreenUserPlan.vue [hunk 4]
```diff
@@ -1,58 +1,76 @@
...
-		<p class="title is-5 has-text-centered">
-			<template v-if="isReturningUser">
-				Twój obecny plan pracy nie ulegnie zmianie.<br>
-				Lekcje będą się otwierały zgodnie z ustalonymi przez Ciebie datami.
-			</template>
-			<template v-else>
-				Lekcje będą się otwierały zgodnie z ustalonymi w planie datami.
-			</template>
-		</p>
+		<div v-if="isPlanBuilderEnabled">
```
Comment (kvas-damian) [inline resources/assets/js/components/onboarding/OnboardingScreenUserPlan.vue]: `<template>`

File: resources/assets/js/components/onboarding/OnboardingScreenUserPlan.vue [hunk 5]
```diff
@@ -1,58 +1,76 @@
...
+			<p class="title is-6 has-text-centered">Na tym kursie nie ma możliwości ustalenia indywidualnego planu pracy.</p>
+			<div class="margin-top-huge">
+				<img
+					:src="imageUrl"
+					alt=""
+					class="onboarding-plan-image"
+				>
+				<div>
+					<p class="margin bottom">🗓 Proponowany przez nas plan pracy trwa od <strong>{{defaultPlanStartDate}}</strong> i zakłada pracę <strong>5&nbsp;dni w tygodniu przez 14 tygodni</strong>. Lekcje otwierają się po jednej każdego dnia. Do raz otwartych lekcji można zawsze powrócić!</p>
+					<p class="margin bottom">💡 Plan zakłada optymalną kolejność przerabiania przedmiotów</p>
```
Comment (kvas-damian) [inline resources/assets/js/components/onboarding/OnboardingScreenUserPlan.vue]: lack of punctuation at the end of text in this line.

File: resources/assets/js/i18n/pl/onboarding.js [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+	final: {
+		ldek: {
+			welcome: `Witaj na kursie do LDEK 🎉`,
+			firstStep: `
+				<h3 class="title is-4">Rozwiąż Wstępny LDEK</h3>
+				<div class="content">
+					<ul>
+						<li>Rozwiąż Wstępny LDEK przed pierwszą lekcją, jest to pierwszy z warunków uzyskania Gwarancji Satysfakcji.</li>
+						<li>Gwarantujemy, że rozwiązanie wstępnego egzaminu wpłynie pozytywnie na Twoją naukę – daj sobie 3-4 godziny na to wyzwanie!</li>
+						<li>Jeżeli realizujesz aktualnie indywidualny plan pracy w ramach przedłużonego dostępu, możesz pominąć wstępny LEK jeśli masz go rozwiązanego.</li>
```
Comment (kvas-damian) [inline resources/assets/js/i18n/pl/onboarding.js]: "indywidualny plan" - we don't allow that.
"LEK"

File: resources/assets/js/i18n/pl/onboarding.js [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+						<li>Wstępny LDEK będzie dla Ciebie zawsze otwarty, znajdziesz go w lekcji WSTĘPNY LDEK.</li>
+					</ul>
+				</div>
+			`,
+			secondStep: `
+				<div>
+					<h3 class="title is-4">Zapoznaj się z modułami Wstęp do kursu i Efektywna nauka</h3>
+					<p>Ukończ moduły "Wstęp do kursu" i "Efektywna nauka" a zrozumiesz metodykę pracy na kursie i oszczędzisz wiele czasu na nauce!</p>
```
Comment (kvas-damian) [inline resources/assets/js/i18n/pl/onboarding.js]: "Efektywna nauka" is a lesson and "Wstęp do kursu" is a group that includes the first. Was it intended to put both here?

File: resources/assets/js/i18n/pl/onboarding.js [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+						<li>Gwarantujemy, że rozwiązanie wstępnego egzaminu wpłynie pozytywnie na Twoją naukę – daj sobie 3-4 godziny na to wyzwanie!</li>
+						<li>Jeżeli realizujesz aktualnie indywidualny plan pracy w ramach przedłużonego dostępu, możesz pominąć wstępny LEK jeśli masz go rozwiązanego.</li>
+						<li>Wstępny LEK będzie dla Ciebie zawsze otwarty, znajdziesz go w module WSTĘPNY LEK.</li>
+					</ul>
+				</div>
+			`,
+			secondStep: `
+				<div>
+					<h3 class="title is-4">Zapoznaj się z modułami Wstęp do kursu i Efektywna nauka</h3>
+					<p>Ukończ moduły "Wstęp do kursu" i "Efektywna nauka" a zrozumiesz metodykę pracy na kursie i oszczędzisz wiele czasu na nauce!</p>
```
Comment (kvas-damian) [inline resources/assets/js/i18n/pl/onboarding.js]: same as on ldek

File: resources/assets/js/components/onboarding/OnboardingScreenTutorial.vue (hunk 3 above)
Comment (adamkarminski) [reply resources/assets/js/components/onboarding/OnboardingScreenTutorial.vue]: Nope :)

Comment (kvas-damian) [reply resources/assets/js/components/onboarding/OnboardingScreenTutorial.vue]: I was thinking about dedicated video for ldek

**Comment by kvas-damian** [comment]
Look at travis output - we have a lot of eslint rules enabled :P

File: resources/assets/js/i18n/pl/onboarding.js [hunk 9]
```diff
@@ -0,0 +1,109 @@
+export const onboarding = {
+	learningStyle: {
+		ldek: {
+			title: '4 sposoby nauki',
+			time1: 'Średnio 3 godziny dziennie',
+			time2: 'Średnio 4 godzin dziennie',
```
Comment (rogatty) [inline resources/assets/js/i18n/pl/onboarding.js]:
```suggestion
			time2: 'Średnio 4 godziny dziennie',
```

File: resources/assets/js/i18n/pl/onboarding.js [hunk 10]
```diff
@@ -0,0 +1,109 @@
...
+		default: {
+			welcome: `Witaj na kursie do LEK 🎉`,
+			firstStep: `
+				<h3 class="title is-4">Rozwiąż Wstępny LEK</h3>
+				<div class="content">
+					<ul>
+						<li>Rozwiąż Wstępny LEK przed pierwszą lekcją, jest to pierwszy z warunków uzyskania Gwarancji Satysfakcji.</li>
+						<li>Gwarantujemy, że rozwiązanie wstępnego egzaminu wpłynie pozytywnie na Twoją naukę – daj sobie 3-4 godziny na to wyzwanie!</li>
+						<li>Jeżeli realizujesz aktualnie indywidualny plan pracy w ramach przedłużonego dostępu, możesz pominąć wstępny LEK jeśli masz go rozwiązanego.</li>
+						<li>Wstępny LEK będzie dla Ciebie zawsze otwarty, znajdziesz go w module WSTĘPNY LEK.</li>
```
Comment (rogatty) [inline resources/assets/js/i18n/pl/onboarding.js]:
```suggestion
						<li>Wstępny LEK będzie dla Ciebie zawsze otwarty, znajdziesz go w lekcji WSTĘPNY LEK.</li>
```

**Pull request merged**